colon-app/src/main/java/pl/edu/agh/to/app/controller/HomeController.java

```diff
@@ -1,17 +1,27 @@
 package pl.edu.agh.to.app.controller;
```

Comment:
Or maybe you could merge your branch with master, so I can merge master with my branch, and then make pull request? I don't know which option is more gitty. :)